Toggl.Foundation.Tests/MvvmCross/ViewModels/TimeEntriesViewModelTests.cs

```diff
@@ -330,9 +330,9 @@ public async ThreadingTask ShowsTheUndoUI()
             [Fact]
             public async ThreadingTask DoesNotHideTheUndoUITooEarly()
             {
-                var observable = viewModel.DelayDeleteTimeEntry.Execute(timeEntry);
+                viewModel.DelayDeleteTimeEntry.Execute(timeEntry);
                 SchedulerProvider.TestScheduler.AdvanceBy(Constants.UndoTime.Ticks - 1);
-                await observable;
+                await ViewModel.DelayDeleteTimeEntry.Elements;;
```

Comment:
Two ; 👀

Reply:
Come on, help the guy with the

```suggestion
                await ViewModel.DelayDeleteTimeEntry.Elements;
```